Components/banner.js

```diff
@@ -1,2 +1,95 @@
 import React from 'react';
+import { makeStyles, useTheme } from '@material-ui/core/styles';
+import MobileStepper from '@material-ui/core/MobileStepper';
 import Button from '@material-ui/core/Button'
+import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft';
+import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight';
+import SwipeableViews from 'react-swipeable-views';
+import { autoPlay } from 'react-swipeable-views-utils';
+import Box from '@material-ui/core/Box';
+
+const AutoPlaySwipeableViews = autoPlay(SwipeableViews);
+
+// to update banner on home page, change tutorialSteps properties
+const tutorialSteps = [
+  {
+    label: 'The Fellas',
+    imgPath:
+      '/TheFellas.jpg',
+	//href:
```

Comment:
What is this / Do we need it?

Reply:
I guess we don't need it. I assumed we would have to put the path from the component directory but it was being a little weird so I just put `/filename.jpg` but it seems to work without it.